packages/common/src/core/schemas/internal/getCategoryItems.ts

```diff
@@ -17,24 +18,40 @@ export async function getCategoryItems(
   const url = `${hubRequestOptions.portal}/portals/${orgId}/categorySchema`;
   try {
     const { categorySchema } = await request(url, hubRequestOptions);
-    return parseCategories(categorySchema[0].categories, []);
+    // All categories need to be prefixed with "/Categories" to be valid
+    return convertCategoryProps(categorySchema[0].categories, "/Categories");
   } catch (e) {
     return [];
   }
 }
 
-function parseCategories(
-  categories: any[],
-  allCategories: any[]
+/**
+ * Convert the categories into a format that can be used
+ * by the IUiSchemaComboboxItem interface
+ *
+ * @param obj The object to convert
+ * @param parentValue The parent value to append to the value
+ * @returns the converted object where `title` becomes `value` and `categories` becomes `children`
+ */
+function convertCategoryProps(
+  arrayOfCategories: any,
+  parentValue: string
 ): IUiSchemaComboboxItem[] {
-  categories.forEach((c) => {
-    if (!c.categories?.length) {
-      allCategories.push({
-        value: c.title,
-      });
-    } else {
-      parseCategories(c.categories, allCategories);
-    }
+  return arrayOfCategories.map((category: any) => {
+    // if there is a parentValue, append the category title to it to give it a unique value
+    // that can be distinguished from other categories
+    const value = `${parentValue}/${category.title}`;
+
+    // use just the title as the label, so we don't have redundant information visually
+    const label = category.title;
+
+    // if there are no more categories, return null for children, otherwise recursively call this function
+    return {
+      value,
+      label,
+      children: !category.categories?.length
+        ? []
+        : convertCategoryProps(category.categories, value),
```

Comment:
Small stylistic thing with ternaries, it's usually best to have the expected (happy) path as the first condition and have the fallback as the second condition. e.g:
```
!!category.categories?.length
        ? convertCategoryProps(category.categories, value)
        : []
```

This parallels what we usually do with the logical OR. e.g.:
`return obj.myField || [];`